samples/dsm/akeneo-dsm/back-end/controllers/api/pim.controller.js

```diff
@@ -1,5 +1,7 @@
-const { pim_instance } = require('../../use-cases/api/pim-instance');
+const { pimInstance } = require('../../use-cases/api/pim-instance');
+const { pimProductsRetrieve } = require('../../use-cases/api');
```

Comment:
I don't know anything about js node so I am curious. Why you use ../../use-cases/api which point to a folder for pimProductsRetrieve while pimInstance point to a specific file ../../use-cases/api/pim-instance